nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ScanAttribute.java

```diff
@@ -60,13 +64,21 @@
 @Tags({"scan", "attributes", "search", "lookup"})
 @CapabilityDescription("Scans the specified attributes of FlowFiles, checking to see if any of their values are "
         + "present within the specified dictionary of terms")
+@WritesAttributes({
+        @WritesAttribute(attribute = "dictionary.hit.{n}.attribute", description = "The attribute name that had a value hit on the dictionary file."),
```

Comment:
sorry for not thinking of this earlier but i think we should use 'dictionary.hit.attribute.{n}' for these three attributes we add.  This makes it easier for downstream users of these attributes to select attributes of interest because they can say things like 'if any attribute starts with dictionary.hit.attribute with a value of foo'.

If you agree please do change that.